scripts/publish-site.sh

```diff
@@ -12,7 +13,7 @@ cd "$1" || exit
 yarn
 
 echo "=== Copying built Gatsby to website."
-gatsby-dev --scan-once
+gatsby-dev --scan-once --packages gatsby gatsby-transformer-documentationjs
```

Comment:
specific reason to scope to these packages? Might be handy to add a comment

Reply:
I will remove this change before actual merge (see TO-DO list in PR description) - this is part of temporary commit that allow me to actually build preview on netlify (this + removing bunch of pages in www/gatsby-node.js)